### PR TITLE
rubby:0.8.0

### PR DESCRIPTION
**Pull request by Andrew15-5**
I am submitting
- [x] a new package
- [ ] an update for a package

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

Description: Library provides with function to create ruby (furigana) next to base text.
Since there is no built-in functionality for ruby in Typst (yet) this package could be very helpful until official support arrives.

### CODE REVIEW

**Comment by reknih** [comment]
Hey, thanks for your package submission! Could you add a bit of "pazazz" to the package name? As-is, it's merely descriptive, which could deceive users into assuming this is an officially endorsed package.

**Comment by Andrew15-5** [comment]
> Could you add a bit of "pazazz" to the package name?

Oh, no, anything but this. It's like naming variables level 100. Ok, I hope I'll come up with something in a few hours.

**Comment by Andrew15-5** [comment]
Ha-ha, does "rubby" have enough "pazazz" to be merged? ;)

**Comment by reknih** [comment]
That would work!

**Comment by Andrew15-5** [comment]
@reknih, I renamed it.

File: packages/preview/rubby-0.8.0/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# rubby (Typst package)
+
+## Usage
+
+```typst
+#import "@preview/rubby:0.8.0": get_ruby
+
+ruby = get_ruby(
```
Comment (laurmaedje) [inline packages/preview/rubby-0.8.0/README.md:8]:
```suggestion
#let ruby = get_ruby(
```

File: packages/preview/rubby-0.8.0/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  size: 0.5em,         // Ruby font size
+  dy: 0pt,             // Vertical offset of the ruby
+  pos: top,            // Ruby position (top or bottom)
+  alignment: "center", // Ruby alignment ("center", "start", "between", "around")
+  delimeter: "|",      // The delimeter between words
```
Comment (laurmaedje) [inline packages/preview/rubby-0.8.0/README.md:13]:
```suggestion
  delimiter: "|",      // The delimiter between words
```

File: packages/preview/rubby-0.8.0/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+#ruby[ふりがな][振り仮名]
+
+Treat each kanji as a separate word:
+#ruby[とう|きょう|こう|ぎょう|だい|がく][東|京|工|業|大|学]
+```
+
+If you don't want automatically wrap text with delimiter:
+
+```typst
+ruby = get_ruby(auto_spacing: false)
```
Comment (laurmaedje) [inline packages/preview/rubby-0.8.0/README.md:27]:
```suggestion
#let ruby = get_ruby(auto_spacing: false)
```

**Review by laurmaedje — COMMENTED** [review]
Found some small issues, otherwise it looks good!

**Comment by Andrew15-5** [comment]
Yes, it completely slipped before my eyes. I did rush it a bit, huh... Thanks for noticing!

**Comment by laurmaedje** [comment]
Thank you!

**Pull request merged**